lib/elixir/lib/enum.ex

```diff
@@ -1653,6 +1653,9 @@ defmodule Enum do
   It assumes that the sample being returned can fit into memory;
   the input `enumerable` doesn't have to, as it is traversed just once.
 
+  If a range is passed into the function it will not be expanded so it
```

Comment:
Hrm, the issue is that we are not saying what it is doing, we are saying what it is **not doing**, and that doesn't reveal much. What if we try something like: "If a range is passed into the function, Elixir will pick a random value between the range limits, without traversing the whole range."